src/opt_handlers.c

```diff
@@ -2811,6 +2821,24 @@ trashdir_handler(OPT_OP op, optval_t val)
 	free(expanded_path);
 }
 
+static void
+trash_filefmt_zero_handler(OPT_OP op, optval_t val)
+{
+	if (cfg.trash_filefmt_zero != NULL)
+		free(cfg.trash_filefmt_zero);
```

Comment:
By the way, all C standards explicitly allow argument of `free()` to be `NULL`, so such checks are never necessary.